src/main/java/com/uid2/shared/store/IServiceLinkStore.java

```diff
@@ -0,0 +1,11 @@
+package com.uid2.shared.store;
+
+import com.uid2.shared.model.ServiceLink;
+
+import java.util.Collection;
+
+public interface IServiceLinkStore {
```

Comment:
I am wondering if we need another API here to fetch a record with a combination of <service_id, link_id>. In operator when a request comes in to /identity/map, the operator will validate the request by checking if the <service_id, link_id> combination exists. Would it be better to provide that API here or fetch all service links and then filter by service_id ? 
Your thoughts @Ian-Nara @thomasm-ttd

Reply:
I think it is okay to just get all and filter. I have implemented a similar thing for listing all links by site id, And this search by site id functionality also exists for the keys UI. If we want to update some fields in a service_link though we probably want this anyway.